modules/tensor_mechanics/src/materials/ComputeIsotropicElasticityTensor.C

```diff
@@ -124,6 +126,9 @@ ComputeIsotropicElasticityTensor::ComputeIsotropicElasticityTensor(
 void
 ComputeIsotropicElasticityTensor::computeQpElasticityTensor()
 {
+  // Let the Stress Calculator classes know that this class only has constant values
+  _elasticity_tensor_is_constant[_qp] = true;
+
```

Comment:
After talking with @dschwen, I've moved this material property from initQpStatefulProperties, since it is not intended to be stateful anyway, to the main compute method to ensure that inheriting classes overwrite the setting of the `_elasticity_tensor_is_constant` flag.  It's definitely not the most computationally efficient place to set this value, so please let me know if there's somewhere else I should set the value.